app/controllers/planning_applications/assessment/local_policy_areas_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class LocalPolicyAreasController < AuthenticationController
+      before_action :set_planning_application
+      before_action :set_local_policy, only: %i[show edit update new destroy]
+      before_action :set_local_policy_area, only: %i[edit show]
+
+      def show
+      end
+
+      def new
+        @local_policy_area = @local_policy.local_policy_areas.new
+      end
+
+      def edit
+      end
```

Comment:
Sorry, wasn't clear - we should be using `respond_to` blocks in all actions

Reply:
Apologies - updated.